fs/base.py

```diff
@@ -1645,6 +1645,9 @@ def match(self, patterns, name):
             patterns (list, optional): A list of patterns, e.g.
                 ``['*.py']``, or `None` to match everything.
             name (str): A file or directory name (not a path)
+            accept_prefix (bool): If ``True``, the name is
```

Comment:
Could you maybe add an example to the **Example** section below showing what the `accept_prefix` does? I think it'd be easier to understand that way.